src/main/resources/freemarker/es7x/mapping/index-template-log-v4.ftl

```diff
@@ -0,0 +1,79 @@
+<#ftl output_format="JSON">
+{
+    "index_patterns": ["${indexName}*"],
+    "settings": {
+        <#if indexLifecyclePolicyLog??>"${indexLifecyclePolicyPropertyName}": "${indexLifecyclePolicyLog}",</#if>
+        <#if indexLifecyclePolicyLog??>"index.lifecycle.rollover_alias": "${indexName}",</#if>
+        "index.number_of_shards":${numberOfShards},
+        "index.number_of_replicas":${numberOfReplicas},
+        "index.refresh_interval": "${refreshInterval}"
+        <#if extendedSettingsTemplate??>,<#include "/${extendedSettingsTemplate}"></#if>
+    },
+    "mappings": {
+        "properties": {
+            "@timestamp": {
+                "type": "date"
+            },
+            "api-id": {
+                "type": "keyword"
+            },
+            "request-id": {
+                "type": "keyword"
+            },
+            "client-identifier": {
+                "type": "keyword"
+            },
+            "request-ended": {
+                "type": "boolean"
+            },
+            "entrypoint-request": {
+                "type": "object",
+                "properties": {
+                    "body":{
```

Comment:
The `uri` and `method` are missing. I'm wondering if we should add them to the template 🤔

Reply:
Don't know, not present in v2 so I just did the same thing.